doc/classes/PhysicsServer2D.xml

```diff
@@ -935,13 +935,28 @@
 				Creates a 2D space in the physics server, and returns the [RID] that identifies it. A space contains bodies and areas, and controls the stepping of the physics simulation of the objects in it.
 			</description>
 		</method>
+		<method name="space_flush_queries" is_experimental="true">
```

Comment:
Do we really need these to be experimental?

Reply:
This PR has not been extensively tested, and I would rather mark all added APIs as experimental to reserve the right to break the compatibility by revoking.